starboard/android/shared/audio_track_audio_sink_type.h

```diff
@@ -77,6 +77,8 @@ class AudioTrackAudioSinkType : public SbAudioSinkPrivate::Type {
   }
 
   void Destroy(SbAudioSink audio_sink) override {
+    // Use audio_sink.flush() instead of re-creating a new audio_sink.
+    // Tracking bug: b/330793785
```

Comment:
nit: usually we would write it in a format like: `// TODO (b/330793785): Use audio_sink.flush() instead of re-creating a new audio_sink.`

Reply:
Done.